### PR TITLE
Add sorting and refresh card style on plugin hub

### PR DESCRIPTION
**Pull request by nijikokun**
**Note** there is an update to the social card component (adds key support) to prevent this issue from occurring after the attempt to update to `6.0.0-beta` failed https://github.com/nfl/react-helmet/issues/373 - 90% of the time spent on this PR went to this.

---

Can sort by the following:

- Downloads (Yearly -- total in our sense) (Default)
- Name
- Release Date

Order by:

- Ascending
- Descending (Default)

Example:

Screen Shot 2020-03-23 at 10 03 19 AM

### CODE REVIEW

**Comment by nijikokun** [comment]
Looks like netlify cannot fetch the `npm` urls...

File: src/components/social-cards.js [hunk 1]
```diff
@@ -1,34 +1,44 @@
 import React from 'react';
 import Helmet from 'react-helmet';
 
-const SocialCards = ({isBanner, title, summary}) => (
+const SocialCards = ({ isBanner, title, summary }, key) =>
```
Comment (develohpanda) [inline src/components/social-cards.js]: Shouldn't `key` be part of the prop object here? 🤔 

```suggestion
const SocialCards = ({ isBanner, title, summary, key }) =>
```

Comment (nijikokun) [reply src/components/social-cards.js]: It could be, its optional

**Comment by nijikokun** [comment]
Fixed fetching the `npm` downloads, appears that when a library is very new, and you attempt to fetch individually (we do this for libraries that are namespaced e.g. contain an `@` symbol, because they don't work with the bulk query) it returns a 404. This now is handled by returning zero downloads instead.

**Comment by nijikokun** [comment]
Made changes suggested by Opender, lead to having to change SocialCards implementation from JSX style to Function style.

Changed style:

Screen Shot 2020-03-31 at 12 45 34 PM

**Comment by nijikokun** [comment]
> LGTM!
> 
> Was `key` a required prop needed for the newer version of `Helmet`? Just trying to understand what drove that change 😄

Not required for the newer version, but to prevent a recursive deep-equals check when mutations occur.

**Pull request merged**